plenum/server/replica.py

```diff
@@ -1433,8 +1444,8 @@ def _apply_pre_prepare(self, pre_prepare: PrePrepare):
                                                            state_root=self.stateRootHash(pre_prepare.ledgerId,
                                                                                          to_str=False),
                                                            txn_root=self.txnRootHash(pre_prepare.ledgerId,
-                                                                                     to_str=False)
-                                                           )
+                                                                                     to_str=False),
+                                                           primaries=self.node.primaries)
```

Comment:
We must call `get_primaries_for_view_no` here since we can receive a PP from a previous view.